.github/workflows/error-ref-publisher.yaml

```
@@ -0,0 +1,63 @@
+name: Meshkit Error Codes Utility Runner
+on:
+  push:
+    branches:
+      - 'master'
+    paths:
+      - '**.go'
+
+jobs:
+  Update-error-codes:
+    name: Error codes utility
+    if: github.repository == 'layer5io/meshkit'
```

```suggestion
    if: github.repository == 'meshery/meshery-istio'
```
Its the org/name of this repo : )

---

Oh so sorry! I honestly thought there's no changes needed. I haven't done these kinds of feature before. Noted!

---

No worries. :grin: